### ISSUE TITLE
Let's fork the project! 

### ISSUE THREAD

**Issue by LifeIsStrange** [sweet-js/sweet-core#759]
Last commit is from 2 years ago...
Having macros in javascript and typescript is awesome, and we are the most popular programming language in the world!
Aren't we capable of forking and maintaining this fundamental project?
Any volunteers?

**Comment by purefan** [comment]
I dont think you need anyone's permission, just do it yourself and if its promotion what you want go to gitter and open a channel. 2 years of no activity or forks make me think there's little interest but I've been wrong before

**Comment by LifeIsStrange** [comment]
@purefan sadly I have not the time nor the skill to fork this project.

**Comment by HeatherSoron** [comment]
I'm somewhat tempted to fork it myself, honestly. I really want Lisp-like macros in JS, but #760 is a pretty big downside for me. Luckily, this is BSD 2-clause, which removes basically all non-technical impediments :).

If I do fork it and make meaningful patches, I might drop a link here.

**Comment by simplygreatwork** [comment]
I've been dabbling with creating a macro system for WebAssembly WAT text files in my spare time: https://github.com/simplygreatwork/sweet-webassembly

It's not exactly Sweet.js but I do really like Sweet.js as an inspiration. What I have built so far is not very end user friendly at this point. One task I just finished building is making the parser indentation sensitive so you could use sexpressions (braces) or indentation to define your code. Lots more work to be done. For example there's no API yet for configuring macros dynamically/externally.
